GG/GG/Font.h

```diff
@@ -206,6 +209,14 @@ class GG_API Font
 
         virtual ~TextElement(); ///< Virtual dtor.
 
+        /** Rebind to \p whole_text, by binding the SubString data member text to \p whole_text.
+
+            Binding to a new \p whole_text is very fast compared to re-parsing the entire \p
+            whole_text and allows TextElements of TextElementType TEXT to be changed quickly if it
+            is known that the parse would be the same.
```

Comment:
I've expanded the comment for the Bind function to add use cases.

I also added a copy constructor and an assignment constructor in the PR #989 where the constructor that uses the results of `TextAndElementsAssembler` is added.  This is the second obvious use case for `Bind()`.

The operation of adding/removing tags like `<u>`, was not something that I added. It did not impact the performance of the `ListWnd` `FleetWnd` which was the focus of this set of PRs.  This functionality would be the counterpart to `ChangeTemplatedText`, but it doesn't exist.

I also made the requested style changes.